lambda/lambda_function.py

```diff
@@ -4,17 +4,17 @@
 # Please visit https://alexa.design/cookbook for additional examples on implementing slots, dialog management,
 # session persistence, api calls, and more.
 
-import logging
-import ask_sdk_core.utils as ask_utils
 import json
+import logging
 
-from ask_sdk_core.skill_builder import SkillBuilder
-from ask_sdk_core.dispatch_components import AbstractRequestHandler
-from ask_sdk_core.dispatch_components import AbstractExceptionHandler
+import ask_sdk_core.utils as ask_utils
+from ask_sdk_core.dispatch_components import (
```

Comment:
Would it be possible to lint but keep single module per line imports?
This is not critical but in general doing a single module import per line makes merge conflict resolution simpler

Reply:
This one isn't so easy - I think it's a limitation of how Black groups code

Reply:
Reading on the Black website...

> The following notable open-source projects trust Black with enforcing a consistent code style: pytest, tox, Pyramid, Django, Django Channels, Hypothesis, attrs, SQLAlchemy, Poetry, PyPA applications (Warehouse, Bandersnatch, Pipenv, virtualenv), pandas, Pillow, Twisted, LocalStack, every Datadog Agent Integration, Home Assistant, Zulip, Kedro, OpenOA, FLORIS, ORBIT, WOMBAT, and many more.
> 
> The following organizations use Black: Facebook, Dropbox, KeepTruckin, Mozilla, Quora, Duolingo, QuantumBlack, Tesla, Archer Aviation.

I think this are enough amazing examples for me to drop my initial request 😄 Let's use Black and see how it works for us.
Fyi: I read the [code style here](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#pragmatism), quite good page explaining the reasonings behind all choices. Pragmatism is the main word, and that feels good to me 👍